stdlib/2/email/charset.pyi

```diff
@@ -2,6 +2,10 @@ def add_charset(charset, header_enc=..., body_enc=..., output_charset=...) -> No
 def add_alias(alias, canonical) -> None: ...
 def add_codec(charset, codecname) -> None: ...
 
+QP: int
+BASE64: int
+SHORTEST: int
```

Comment:
Could you add `# undocumented` comments to these lines? (I know they are missing from the Python 3 stub, but that's because I missed it.)

Reply:
Done, see #2857.